test/cli/run/cjs-repro/c.js

```diff
@@ -0,0 +1,8 @@
+import EE from "node:events";
+
+console.log([EE]);
+
+var __commonJS = (cb, mod) => function () {};
```

Comment:
FWIW, I think this CommonJS function is not used and therefore has no impact

Reply:
oh oops i never meant to commit this file